docs/api/session.md

```diff
@@ -69,6 +69,59 @@ console.log(ses.getUserAgent())
 
 The following events are available on instances of `Session`:
 
+#### Event: 'bluetooth-pair-prompt' _Windows_ _Linux_
+
+Returns:
+
+* `event` Event
+* `details` Object
+  * `deviceId` string
+  * `pairingKind` string - The type of pairing prompt being requested.
+     Value will be one of `confirm`, `confirmPin`, or `providePin`.
+    * `confirm`
+      This prompt is requesting confirmation that the bluetooth device should
+      be paired.
+    * `confirmPin`
+      This prompt is requesting confirmation that the provided pin matches the
+      pin displayed on the device.
+    * `providePin`
+      This prompt is requesting that a pin be provided for the device.
+  * `frame` [WebFrameMain](web-frame-main.md)
+  * `pin` string - If the `pairingKind` is `confirmPin`, this value will be
+     the pin value to verify.
+* `callback` Function
+  * `confirmed` boolean - `false` should be passed in if the dialog is canceled.
+    If the `pairingKind` is `confirm` or `confirmPin`, this value should indicate
+    if the pairing is confirmed.  If the `pairingKind` is `providePin` the value
+    should be `true` when a value is provided.
+  * `pin` string | null (optional) - When the `pairingKind` is `providePin`
+    this value should be the required pin for the bluetooth device.
+
+Emitted when a bluetooth device requires a response to pairing.  This event
+allows developers to handle devices that require additional validation for
+pairing.  When handling this event, `event.preventDefault()` should be called
+to prevent the default behavior of cancelling this prompt.
```

Comment:
Requiring `preventDefault()` seems a little inelegant... though it would be annoying to check for whether there are any listeners, it seems like the behavior of "cancel this event unless there are any listeners, in which case the event handler is responsible for calling the callback" is fairly clear.

For that matter... perhaps this would be a good candidate for a `set*Handler` pattern, a la `setPermissionHandler` / `setDisplayMediaRequestHandler` etc.?

Reply:
@nornagon hmm yeah maybe something like `setBluetoothPairingHandler`.  I agree that `preventDefault()` is a little janky.  I'll look at refactoring to instead use the `set*Handler` pattern.